.expeditor/end_to_end.pipeline.yml

```diff
@@ -278,6 +278,17 @@ steps:
             - BUILD_PKG_TARGET=x86_64-windows
             - BUILDKITE_AGENT_ACCESS_TOKEN
 
+  - label: "[:linux: test_pkg_download]"
+    command:
+      - .expeditor/scripts/end_to_end/setup_environment.sh DEV
```

Comment:
Need to use "dev" rather than "DEV"; see #7100 for details

Reply:
Fixed up in https://github.com/habitat-sh/habitat/pull/7104/commits/2da2907b008d5d7bcfae15e15cf6504fbbb5d734